production/helm/loki/templates/read/deployment-read.yaml

```diff
@@ -54,7 +54,7 @@ spec:
         {{- toYaml .Values.loki.podSecurityContext | nindent 8 }}
       terminationGracePeriodSeconds: {{ .Values.read.terminationGracePeriodSeconds }}
       containers:
-        - name: read
+        - name: loki
```

Comment:
Why is this change needed?

Reply:
There's a few queries in our dashboards that are looking for `container="loki"`